functions.opengraph.php

```diff
@@ -336,20 +348,12 @@ function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) {
 
 	// Third fall back, Core Site Icon, if valid in size. Added in WP 4.3.
 	if ( empty( $image ) && ( function_exists( 'has_site_icon') && has_site_icon() ) ) {
-		$max_side = max( $width, $height );
-		$image_url = get_site_icon_url( $max_side );
-
 		$img_width  = '';
 		$img_height = '';
-		$cached_image_id = get_transient( 'jp_' . $image_url );
-
-		if ( false !== $cached_image_id ) {
-			$image_id = $cached_image_id;
-		} else {
-			$image_id = attachment_url_to_postid( $image_url );
-			set_transient( 'jp_' . $image_url, $image_id );
-		}
 
+		$max_side = max( $width, $height );
+		$image_url = get_site_icon_url( $max_side );
+		$image_id = get_option( 'site_icon' );
```

Comment:
Since we check for `has_site_icon()` before to start the whole process, I don't think that's necessary. For `has_site_icon()` to return `true`, `get_option( 'site_icon' )` has to return an ID.